src/contracts/deposits-registry/sanity-checker/integrity-checker/integrity-checker.service.ts

```diff
@@ -62,7 +59,7 @@ export class DepositIntegrityCheckerService {
   public async checkLatestRoot(
     blockNumber: number,
     eventsCache: VerifiedDepositEvent[],
-  ): Promise<void> {
+  ): Promise<boolean> {
```

Comment:
Checks the integrity of the latest root -> Verifies/Check the integrity of the latest root of the local deposit events tree (combining cached and fresh events) against the blockchain deposit root for a specified block number.

the same below for finalized